src/main/java/com/quizapp/enterprise/services/QuizService.java

```diff
@@ -34,19 +37,24 @@ public Quiz getQuiz(int quizId) throws Exception {
 
         Optional<Quiz> quizToReturn =  quizRepository.findById(quizId);
         if(!quizToReturn.isPresent()){
-            throw new Exception("Quiz with id "  + quizId +  "does not exist");
-
+            StringBuilder missingQuizException = new StringBuilder("Quiz with id ")
+                    .append(quizId)
+                    .append("does not exist");
+            throw new Exception(missingQuizException.toString());
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/